front/src/modules/project/components/AddOrEditProjectModal.tsx

```diff
@@ -55,8 +58,14 @@ export default function AddOrEditProjectModal({
   project,
   getProject,
 }: AddOrEditProjectModalProps) {
-  const { t } = useTranslation('operationalStudies/project');
-  const { closeModal } = useContext(ModalContext);
+  const { t } = useTranslation(['operationalStudies/project', 'home']);
```

Comment:
home isn't used anymore

```suggestion
  const { t } = useTranslation('operationalStudies/project');
```

Reply:
fix